cmd/captplanet/run.go

```diff
@@ -43,6 +45,7 @@ type Satellite struct {
 	Repairer    repairer.Config
 	Audit       audit.Config
 	StatDB      statdb.Config
+	Accounts    satellite.Config
```

Comment:
Let's call it something else, not sure yet what. Maybe `Frontend`, `API` or `Gateway` or something like it. As I understand it will be responsible for more than accounts?